core/src/main/java/com/netflix/conductor/core/execution/tasks/Goto.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.conductor.core.execution.tasks;
+
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.Task.Status;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.core.execution.WorkflowExecutor;
+
+/**
+ * @author Shivam
+ *
+ */
+public class Goto extends WorkflowSystemTask {
+	
+	public Goto() {
+		super("GOTO");
+	}
+	
+	@Override
+	public boolean execute(Workflow workflow, Task task, WorkflowExecutor provider) {
+		task.setStatus(Status.COMPLETED);
```

Comment:
We can store more information as a part of goto task output.

Reply:
Please suggest what other information we can store as part of gotoTask. 
The next Task of GOTO is already defined in workflow  DSL.